packages/mds-agency/api.ts

```diff
@@ -113,7 +113,7 @@ function api(app: express.Express): express.Express {
           // stash provider_id
           res.locals.provider_id = provider_id
 
-          log.info(providerName(provider_id), req.method, req.originalUrl)
+          // log.info(providerName(provider_id), req.method, req.originalUrl)
```

Comment:
Should just delete

Reply:
well, we had it before, we might want it again later
generally though I agree